### PR TITLE
Provide additional admin email notifications

### PR DESCRIPTION
**Pull request by bmfmancini**
Add additional admin email alerts

### CODE REVIEW

File: cactid.php [hunk 1]
```diff
@@ -41,6 +41,7 @@ function sig_handler($signo) {
 	case SIGTERM:
 	case SIGINT:
 		cacti_log('WARNING: Cacti Daemon PID[' . getmypid() . '] Terminated on Device[' . gethostname() . ']', true, 'CACTID');
+                admin_email(__('Cacti System Warning'), __('WARNING: Cacti Daemon PID[' . getmypid() . '] Terminated on Device[' . gethostname() . ']', true, 'CACTID'));
```
Comment (netniV) [inline cactid.php:44]: You have used spaces instead of tabs.

File: CHANGELOG [hunk 2]
```diff
@@ -8,6 +8,7 @@ Cacti CHANGELOG
 -issue#4381: Fatal Errors can occur if you are not in a transaction and you attempt to commit one
 -issue#4383: Fixes error callbacks for php 8.0+
 -feature#4344: Add a 15 minute poller interval to global_arrays.php
+-feature#4385 Add additional admin email notifications
```
Comment (netniV) [inline CHANGELOG:11]: Missing colon :)

File: poller.php [hunk 3]
```diff
@@ -797,6 +797,7 @@ function sig_handler($signo) {
 		}
 	} else {
 		cacti_log('WARNING: Cacti Polling Cycle Exceeded Poller Interval by ' . ($loop_end-$loop_start-$poller_interval) . ' seconds', true, 'POLLER', $level);
+                admin_email(__('Cacti System Warning'), __('WARNING: Cacti Polling Cycle Exceeded Poller Interval by ' . ($loop_end-$loop_start-$poller_interval) . ' seconds', true, 'POLLER', $level));
```
Comment (netniV) [inline poller.php:800]: You have used spaces instead of tabs.

File: poller_automation.php [hunk 4]
```diff
@@ -50,9 +50,12 @@
 		db_force_remote_cnn();
 	} else {
 		cacti_log('WARNING: Main Cacti database offline or in recovery.  Can not run automation', false, 'AUTOM8');
+                admin_email(__('Cacti System Warning'), __('WARNING: Main Cacti database offline or in recovery'));
```
Comment (netniV) [inline poller_automation.php:53]: You have used spaces instead of tabs.

File: poller_automation.php [hunk 5]
```diff
@@ -50,9 +50,12 @@
...
+	}
+
 		exit(1);
 	}
-}
+#}
```
Comment (netniV) [inline poller_automation.php:58]: Is there a reason for this comment character ?

File: poller_boost.php [hunk 6]
```diff
@@ -307,6 +307,8 @@ function boost_prepare_process_table() {
 			/* if the runtime was exceeded, allow the next process to run */
 			if ($previous_start_time + $max_run_duration < $start_time) {
 				cacti_log('WARNING: Detected Poller Boost Overrun, Possible Boost Poller Crash', false, 'BOOST SVR');
+                                admin_email(__('Cacti System Warning'), __('WARNING: Detected Poller Boost Overrun, Possible Boost Poller Crash', 'BOOST SVR'  ));
```
Comment (netniV) [inline poller_boost.php:310]: You have used spaces instead of tabs.

**Review by netniV — CHANGES_REQUESTED** [review]

**Comment by bmfmancini** [comment]
@netniV  all fixed

**Pull request merged**